example/tester/views.py

```diff
@@ -22,3 +24,14 @@ def get_success_url(self):
 class TestModelDetailView(DetailView):
     template_name = "detail.html"
     model = TestModel
+
+
+class PydanticJSONView(BaseDetailView):
+
+    model = PydanticTestModel
+
+    def render_to_response(self, context, **response_kwargs):
+        # self.object._meta.get_field("nested").pydantic_schema
```

Comment:
Perhaps a use-case for this is if a response doesn't rely on the widget behaviour (e.g. it's only a JSON API response), then it could implement the validation methods on the pydantic model against the context data. Though that is probably beyond the scope of this project since the behaviour seems to be primarily for implementing the React UI.

Reply:
Yes, that's right. What React library gives us is the UI form that is automatically generated. This can be useful if we use pydantic to validate JSON data agains pydantic schema.